src/plugins/blocksmith/components/image-block-modal/CustomizeOptions.jsx

```diff
@@ -51,104 +57,148 @@ const CustomizeOptions = ({
   url,
 }) => (
   <div className="modal-body-blocksmith">
-    <ImageBlockPreview className="my-4" block={block} />
-    {hasCaption && (block.has('caption') || selectedImage.has('description')) && (
-      <p>Caption: {block.get('caption') || selectedImage.get('description')}</p>
-    )}
-    <FormGroup>
-      <ImageAssetPicker
-        multiAssetErrorMessage="Invalid Action: Trying to assign multiple Gallery Block Poster images."
-        isImageSelected={!!selectedImage}
-        isModalOpen={isAssetPickerModalOpen}
-        isDisabled={false}
-        label="Select an image"
-        node={node}
-        onClearImage={handleClearImage}
-        onSelectImage={handleSelectImage}
-        onToggleAssetPickerModal={handleToggleAssetPickerModal}
-      />
-    </FormGroup>
-    <div style={{ maxWidth: '350px', margin: '0 auto' }}>
+      <ImageBlockPreview className="my-4" block={block} />
+      {hasCaption && (block.has('caption') || selectedImage.has('description')) && (
+        <p>Caption: {block.get('caption') || selectedImage.get('description')}</p>
+      )}
       <FormGroup>
-        <Label>Aspect Ratio</Label>
-        <Select
-          onChange={handleChangeAspectRatio}
-          value={!aspectRatio.value ? null : {
-            label: aspectRatio.value.replace('by', ' by '),
-            value: aspectRatio.value,
-          }}
-          options={aspectRatioOptions}
+        <ImageAssetPicker
+          multiAssetErrorMessage="Invalid Action: Can only select one image."
+          isImageSelected={!!selectedImage}
+          isModalOpen={isAssetPickerModalOpen}
+          isDisabled={false}
+          label="Select an image"
+          node={node}
+          onClearImage={handleClearImage}
+          onSelectImage={handleSelectImage}
+          onToggleAssetPickerModal={handleToggleAssetPickerModal}
         />
       </FormGroup>
-      <FormGroup className="d-flex mb-2">
-        <FormGroup check>
-          <Label check>
-            <Checkbox size="sd" id="isNsfw" checked={isNsfw} onChange={handleChangeCheckbox} />
-            is nsfw
-          </Label>
+      <div className="custom-options-container">
```

Comment:
@jalopez13 - for now, remove the `styles.scss` and add styles here:
```suggestion
      <div className="mt-0 mb-0 mr-auto ml-auto" style={{ maxWidth: '350px' }}>
```

Reply:
@arunkarnati 

Removed styles.scss and added the inline styling to div.